src/middlewared/middlewared/plugins/alert.py

```diff
@@ -347,10 +347,18 @@ async def list(self):
         """
 
         as_ = AlertSerializer(self.middleware)
+        classes = (await self.middleware.call("alertclasses.config"))["classes"]
 
         return [
             await as_.serialize(alert)
-            for alert in sorted(self.alerts, key=lambda alert: (alert.klass.title, alert.datetime))
+            for alert in sorted(
+                self.alerts,
+                key=lambda alert: (
+                    -get_alert_level(alert, classes).value,
```

Comment:
typo `-` ?

Reply:
No, I want the highest level first, so CRITICAL (7) should come before INFO (1)